.velocitas.json

```diff
@@ -1,38 +1,22 @@
 {
-    "packages": [
-        {
-            "repo": "https://github.com/eclipse-velocitas/devenv-runtimes.git",
-            "version": "v3.0.0"
-        },
-        {
-            "repo": "https://github.com/eclipse-velocitas/devenv-devcontainer-setup.git",
-            "version": "v2.0.0"
-        },
-        {
-            "repo": "https://github.com/eclipse-velocitas/devenv-github-templates.git",
-            "version": "v1.0.4"
-        }
-    ],
+    "packages": {
+        "devenv-runtimes": "v3.0.0",
+        "devenv-devcontainer-setup": "v2.0.0",
+        "devenv-github-templates": "v1.0.5",
+        "devenv-github-workflows": "v6.0.2"
+    },
     "components": [
-        {
-            "id": "vehicle-signal-interface"
-        },
-        {
-            "id": "devcontainer-setup"
-        },
-        {
-            "id": "runtime-local"
-        },
-        {
-            "id": "github-templates"
-        }
+        "vehicle-signal-interface",
```

Comment:
Without this the runtimes cannot be started (for executing the examples): I think, it is needed to download the vss.json from the COVESA repo. Basically, therefore just the sub-program "download-vspec" would be sufficient to be present, but that's part of the "vehicle-signal-interface" component.

One option would be to move this program/step from there to devcontainer-setup. Or the download-vspec should be separately called from runtimes instead of presuming to be already done before.